sonic_platform_base/sonic_xcvr/fields/public/cmis.py

```diff
@@ -0,0 +1,14 @@
+from ..xcvr_field import NumberRegField
+from .. import consts
+
+class CableLenField(NumberRegField):
+    def __init__(self, name, offset, *fields, **kwargs):
+        kwargs["deps"] = [consts.LEN_MULT_FIELD]
```

Comment:
I don't see any use of passing the kwargs["deps"] to the super class i.e NumberRegField because, NumberRegField.decode() doesn't use decoded_deps. So, this line can be removed?